modules/pam_sepermit/pam_sepermit.c

```diff
@@ -372,12 +372,23 @@ pam_sm_authenticate(pam_handle_t *pamh, int flags UNUSED,
 	char *level = NULL;
 	const char *cfgfile = SEPERMIT_CONF_FILE;
 
+#ifdef VENDORDIR
+	/* If the specified file does not exist, try the vendor file as fallback. */
+	/* This can be overruled again by a user provided file definition (conf=...) */
+	struct stat buffer;
+	if (stat(cfgfile, &buffer) != 0) {
+	  cfgfile = VENDORDIR"/security/sepermit.conf";
+	  pam_syslog (pamh, LOG_INFO,
+		      "cannot read settings from %s. Trying %s", SEPERMIT_CONF_FILE, cfgfile);
+	}
+#endif
+
 	/* Parse arguments. */
 	for (i = 0; i < argc; i++) {
 		if (strcmp(argv[i], "debug") == 0) {
 			debug = 1;
 		}
-		if (strcmp(argv[i], "conf=") == 0) {
+		if (strncmp(argv[i], "conf=", 5) == 0) {
 			cfgfile = argv[i] + 5;
 		}
```

Comment:
It's definitely a bug worth fixing on its own.
Please use `pam_str_skip_prefix` instead.